packages/layouts-css/src/page.scss

```diff
@@ -95,4 +107,61 @@
       }
     }
   }
+
+  &-bottom-bar {
+    grid-area: bottom-bar;
+    height: calc(var(--iui-size-s)*5 + var(--iui-size-l)); //this is the sizes of the two bottom buttons of side nav
+  }
+
+  &-action-bar {
+    background-color: var(--iui-color-background);
+    border-top: 2px solid var(--iui-color-border);
+    margin-inline: auto;
+    padding: var(--iui-size-m);
+    height: inherit;
+    display: grid;
+    gap: var(--iui-size-xs);
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-areas: 'left center right';
+
+    //#region Add env(safe-area-inset-*) for iPhone notch on the left & right sides of content area
+
+    @include small-monitor {
+      padding-inline: unquote('max(calc(var(--iui-size-l) * 2), env(safe-area-inset-right))') unquote('max(calc(var(--iui-size-l) * 2), env(safe-area-inset-left))');
+    }
+
+    @include tablet {
+      padding-inline: unquote('max(var(--iui-size-xl), env(safe-area-inset-right))') unquote('max(var(--iui-size-xl), env(safe-area-inset-left))');
+    }
+
+    @include mobile {
+      padding: unquote('max(var(--iui-size-l), env(safe-area-inset-right))') unquote('max(var(--iui-size-l), env(safe-area-inset-left))');
```

Comment:
You probably need to update padding to match padded content padding. Right now on mobile padding takes almost all available space. 
![image](https://user-images.githubusercontent.com/83585998/235882388-85e21ed5-b010-45ad-9508-de7c399eff96.png)

Reply:
Right yea. I've adjusted it to size s and it looks better now.

Reply:
size m could also work. S also sounds good.